tests/TestCase.php

```diff
@@ -64,4 +65,23 @@ protected function getEnvironmentSetUp($app): void
         $app['config']->set('queue.failed.database', 'mongodb2');
         $app['config']->set('queue.failed.driver', 'mongodb');
     }
+
+    public function skipIfSearchIndexManagementIsNotSupported(): void
+    {
+        try {
+            $this->getConnection('mongodb')->getCollection('test')->listSearchIndexes(['name' => 'just_for_testing']);
+        } catch (ServerException $e) {
+            switch ($e->getCode()) {
+                // MongoDB 6: Unrecognized pipeline stage name: '$listSearchIndexes'
```

Comment:
Does CI only use MongoDB 6.0+? Just curious if this needs to support anything further back.

Reply:
We test from 4.4 to 7.0
https://github.com/mongodb/laravel-mongodb/blob/a257a9fc4aff5138b276268fa8d734186379a4ed/.github/workflows/build-ci.yml#L17-L21